### ISSUE TITLE
implement cards for people, orgenizations and places 

### ISSUE THREAD

**Issue by amitay7** [daat-hamakom/maps#57]
take info fields from exel

**Comment by mushon** [comment]
Design done

**Comment by mushon** [comment]
Changes implemented:
https://invis.io/EXA8Y0F35

**Comment by mushon** [comment]
@josefspr why is this assigned to me? It's already designed.

**Comment by josefspr** [comment]
@mushon questions re cards
1. person: do you mean to display all events on the vertical flow charts. Isn't this redundant 
since it appears on timeline. Better that display only birth/death no ? What about alternative names. 
2. places don't have cover image (we can add field to admin). any case, if no image use default image of leave empty ?

**Comment by mushon** [comment]
1. The idea was to include YOB & YOD but also accommodate an option for major life events as extra fields for persons. This potentially could also be used for places (conquered by Ottomans…)
+I added alt names
2. Add to admin & If no image leave empty.

**Comment by josefspr** [comment]
regarding life events, think its too much to ask researchers, prefer just YOB & YOD

**Comment by mushon** [comment]
that's why it's optional. I think in Herzel's case for example, the date of the First Zionist Congress is crucial, so is in many cases the date of their immigration and so on…

**Comment by josefspr** [comment]
so if you only have YOB & YOD you still draw this timeline ?

**Comment by mushon** [comment]
If we drop it completely we can simply go with YOB-YOD

**Comment by josefspr** [comment]
@mushon can you add title of person to design (Dr., Rabbi etc) ? (only in meta-card)

**Comment by mushon** [comment]
done.
![new-timeline-person-card](https://cloud.githubusercontent.com/assets/27123/25577214/2f8e7f3e-2e6d-11e7-83c8-dd68013cdf2a.png)

**Comment by josefspr** [comment]
since I don't want the title in the search results/cards, shouldn't it be separated from the name ?

**Comment by josefspr** [comment]
@mushon saw you removed handle of metadata card 
<img width="123" alt="screen shot 2017-05-01 at 6 24 12 pm" src="https://cloud.githubusercontent.com/assets/15631729/25583737/c9b63240-2e9b-11e7-9762-37549f3bdd64.png">
how do you open card ?

**Comment by josefspr** [comment]
@mushon missing image in metacard does not look good. Can't we just use the same placeholder color ?

**Comment by mushon** [comment]
It is this button:
![image](https://cloud.githubusercontent.com/assets/27123/25710449/f880cd14-30f4-11e7-80c7-9c5a2cad516e.png)

You can see it in action here [in this sequence](https://invis.io/W8BKOAVDN)

@talbenbasat please note I also cleaned up the InVision folder and to fix a duplication that happened in one of the updates. I think it would be easier for you to navigate now

**Comment by josefspr** [comment]
@mushon so when there is a filter this button opens and closes meta-cards  ?

**Comment by mushon** [comment]
yes.
In addition to clicking the filter title and the timeline name which also opens the meta cards:

**Comment by talbenbasat** [comment]
@mushon 

what do you mean by timeline name?

And the button you mentioned looks like a part of the timeline task,
enlarging the timeline section to the extended mode

**Comment by josefspr** [comment]
@tal timeline name I understand is when we'll have a reduced timeline, so not relevant now.
yes, we'll use the same button but for now it will only open and close the meta-card

**Comment by josefspr** [comment]
@mushon can you take a look at the expand diminish meta-card button in staging, tell us if you 
think it is ok in your opinion ?

**Comment by mushon** [comment]
looking good. Only remove the ``X`` button from next to the expand / contract button

**Comment by mushon** [comment]
One thing though, currently clicking on the project name in the search field closes it while it should only close if we click the ``X``. When we click the name of the project it should open its card.

**Comment by talbenbasat** [comment]
Hi @mushon,

ok deployed.
The x would be visable only when cards-view is displayed and its purpose is to close the card view

**Comment by mushon** [comment]
Exactly. but please note that the ``X`` you implemented is not the right one please use this one instead:
[close-group.svg.zip](https://github.com/daat-hamakom/maps/files/989508/close-group.svg.zip)

**Comment by josefspr** [comment]
@talbenbasat  @mushon  thanks very much to you both
